free-2.md

```diff
@@ -58,12 +72,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this would be a systems-heavy project. A lot of back-end work would have to be done, in order to create a list of wines that are suggested, and to create a way to rank wines based on specific attributes of various foods.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think a good version of such a project would be a massive undertaking. That said, a working (if imperfect) version seems like something that could be doable in the time we have available.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a cool project because I think it wouldn't require too many changes in the language, but would still have enough logical work that would have to be done that it would be significant. The algorithm by which the best wine is determined would be of paramount importance, however.
+
+I think the biggest challenge with this is that there would have to be a massive database of wines or wine categories, and the qualities of these wines (and the foods they'd go well with) would have to be prepopluated / agreed upon by some wine expert(s). This makes the whole nature of the prograem rather subjective which is not ideal.
```

Comment:
As mentioned prior, this is a cool project and if scoped the right way can make a fine DSL project. The one word of warning I have for this project is make sure it doesn't become a database project instead of a DSL project.